data/Crazycoder122.json

```diff
@@ -0,0 +1,19 @@
+{
+   "name":"Krishnendu Dasgupta",
+   "displayStatsPublic":true,
+   "type":"personal",
+   "bio":"Programmer by passion and also a Musician. Trying to solve the toughest Algorithms of life through Music..",
+   "avatar":"https://github.com/Crazycoder122.png",
+   "links":[
+      {
+         "name":"Krishnendu Dasgupta",
```

Comment:
I recommend updating link name to something like, `Twitter`, or `Follow me on Twitter`.

Reply:
Good suggestions 👍 I will merge this PR and improvements to the profile can come in another PR